xml/art-obs-beginners-guide.xml

```diff
@@ -1038,26 +1114,34 @@ Patch0:         &gitproject;_main.diff</screen>
     osc submitreq [OPTIONS] DESTPRJ [DESTPKG]
     osc submitreq [OPTIONS] SOURCEPRJ SOURCEPKG DESTPRJ [DESTPKG]
     -->
-    <screen>&prompt.user2;<command>osc</command> submitpac &obshome1;</screen>
+    <screen>&prompt.user2;<command>osc</command> submitpac</screen>
     <para>
-     &obsa; knows where the original project is located, so you do not have
-     to add the project name here.
+     &obsa; usually knows where the original project is located, so you do
```

Comment:
Please use "osc submitreq" here .... the "submitpac" is actually more or less obsolete meanwhile...

You could also write it that way that "osc submitreq is by default submitting back to the location where you branched from. But it is also possible to specify a different target manualy